activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -391,7 +391,7 @@ def create!(attributes = {}, &block)
       #   person.pets.concat([Pet.new(name: 'Brain'), Pet.new(name: 'Benny')])
       #   person.pets.size # => 5
       def concat(*records)
-        @association.concat(*records)
+        @association.concat(*records) && self
```

Comment:
same here, it should always return self.

Reply:
Looks like the implementation is completely the same with `#<<`, since 

https://github.com/rails/rails/blob/272e64101bb6daaa4bbaf2d1c64f64c6b82ead8e/activerecord/lib/active_record/associations/collection_proxy.rb#L1055-L1059

https://github.com/rails/rails/blob/272e64101bb6daaa4bbaf2d1c64f64c6b82ead8e/activerecord/lib/active_record/associations/collection_association.rb#L115-L116

If we don't intend to behave `#concat` and `#<<` differently (at least the behavior is the same for now), how about just aliasing?

Reply:
```
irb(main):004:0> [:foo] << [:bar]
=> [:foo, [:bar]]
```

While it is different from behavior of Ruby `Array`, I think this behavior is reasonable for record collection which can't contain nested array. +1 for aliasing!